include/souper/Infer/Pruning.h

```diff
@@ -29,33 +29,35 @@ typedef std::function<bool(Inst *, std::vector<Inst *> &)> PruneFunc;
 
 class PruningManager {
 public:
-  PruningManager(Inst *LHS_, std::vector< souper::Inst* >& Inputs_,
-                 unsigned int StatsLevel_, InstContext& IC_,
-                 SMTLIBSolver *SMTSolver_);
+  PruningManager(SynthesisContext &SC_, std::vector< souper::Inst* > &Inputs_,
```

Comment:
do you have a rationale for putting spaces inside the angle brackets? if there's not a specific reason here, then please just be consistent with other code and don't do this

Reply:
It was already there. See the removed line. I didn't want to change the style, so kept the spaces.

Reply:
kill them please